src/Common/src/System/IO/PathInternal.Unix.cs

```diff
@@ -16,6 +16,11 @@ internal static partial class PathInternal
 
         internal static readonly int MaxComponentLength = Interop.Sys.MaxName;
 
+        internal const string ParentDirectoryPrefix = @"../";
+
+        internal static readonly StringComparison DefaultPathComparison = StringComparison.Ordinal;
```

Comment:
Unfortunately I can't use the existing class as it has dependencies on System.IO.FileSystem. Updating with comments.